### PR TITLE
2470 more labels

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: web-client/src/views/TrialSessionWorkingCopy/AddEditCaseNoteModal.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const AddEditCaseNoteModal = connect(
           )}
         >
           <label className="usa-label" htmlFor="case-notes">
-            {'Judge’s Notes'}
+            {'Judge’s notes'}
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionWorkingCopy/AddEditCaseNoteModal.jsx]: I would prefer that string literals NOT be enclosed in braces but instead entered as plain text and not interpolated.

File: web-client/src/views/TrialSessionWorkingCopy/AddEditSessionNoteModal.jsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const AddEditSessionNoteModal = connect(
           )}
         >
           <label className="usa-label" htmlFor="case-notes">
-            {'Judge’s Notes'}
+            {'Judge’s notes'}
```
Comment (kkoskelin) [inline web-client/src/views/TrialSessionWorkingCopy/AddEditSessionNoteModal.jsx]: adjust to be literal, not interpolated.

**Review by kkoskelin — CHANGES_REQUESTED** [review]
some simple changes requested.  My comments about string literals stand for any place in which you discover them within components. :)

File: web-client/src/views/TrialSessionWorkingCopy/AddEditCaseNoteModal.jsx (hunk 1 above)
Comment (matthopson) [reply web-client/src/views/TrialSessionWorkingCopy/AddEditCaseNoteModal.jsx]: Got it.

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**